### DISCUSSION TITLE
You can now respond to user reviews on your Microsoft Edge Add-ons page

### DISCUSSION THREAD

**Discussion by nagachaitanyalokam** [microsoft/MicrosoftEdge-Extensions#18] [Announcements]
The **'Reviews'** feature has been added to the Microsoft Partner Center interface!

Developers, you can reply to user reviews and feedback received on your Microsoft Edge add-on product page.

To learn more, check out the [**Microsoft Edge Add-ons | NEW Feature to manage user reviews on your extension**](https://aka.ms/MSFT-Edge-Addons-Devs-can-respond-to-User-Reviews) blog.

**Comment by NabokD** [comment]
Thank you so much for the feature!
It will be cool add email notifications about new reviews for my extension!

**Reply by nagachaitanyalokam** [reply]
That's a nice idea! We'll look into it.

**Reply by varaduMSFT** [reply]
Added a new issue to track this - https://github.com/microsoft/MicrosoftEdge-Extensions/issues/21

**Comment by carlobril** [comment]
Is there a way to disable reviews or the option to add a response to a review? I just need the stars or nothing at all.

**Reply by infinitepower18** [reply]
Just curious, why would you want to do that?
